src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.security;
```

Comment:
Let's create an additional package for AuthenticationServiceImpl. The package name will be impl, it will locate in the package security.